core-feature-pack/src/main/resources/content/bin/standalone.conf.bat

```diff
@@ -82,3 +82,6 @@ rem # Uncomment and edit to use a custom java.security file to override all the
 rem set "JAVA_OPTS=%JAVA_OPTS% -Djava.security.properties==C:\path\to\custom\java.security"
 
 :JAVA_OPTS_SET
+
+rem # Uncomment to add a Java agent
+rem set "MODULE_OPTS=-javaagent:agent.jar"
```

Comment:
Just for the record regarding these script changes; we are going to need to adapt the specific scripts used for the product conversion as well

Reply:
I guess I didn't realize we had custom scripts to those. Can you point me to them so I can have a look?

Reply:
Hello James, sure: 
https://github.com/jbossas/wildfly-to-product/commits/wf-to-cd

Take a look at "Community to EAP conversion -- Bring in EAP-specific scripts "

Reply:
Done https://github.com/jbossas/wildfly-to-product/pull/46